pilz_trajectory_generation/src/joint_limits_container.cpp

```diff
@@ -100,12 +100,20 @@ bool JointLimitsContainer::verifyVelocityLimit(const std::string &joint_name,
 
 
 bool JointLimitsContainer::verifyPositionLimit(const std::string &joint_name,
-                                                     const double &joint_position) const
-{
-  return (!( hasLimit(joint_name)
-             && getLimit(joint_name).has_position_limits
-             && (joint_position < getLimit(joint_name).min_position
-                || joint_position > getLimit(joint_name).max_position) ) );
+                                                   const double &joint_position) const {
+  bool limit_ok = (!(hasLimit(joint_name)
```

Comment:
This is also a good contribution in terms of verbosity. But can you please separate into an extra pr.